client/trino-cli/src/main/java/io/trino/cli/QueryRunner.java

```diff
@@ -20,6 +20,7 @@
 import io.trino.client.StatementClient;
```

Comment:
Commit message should contain information how concurrency is handled? What is happening when two connections are trying to authenticate.

Is it it possible to logoff using JDBC? Can one connection invalidate token?